client/layout/universal-navbar-footer/index.tsx

```diff
@@ -0,0 +1,362 @@
+										{ translate( 'WordPress.com Support' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/forums/">{ translate( 'WordPress Forums' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/blog/">{ translate( 'WordPress News' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/go/">{ translate( 'Website Building Tips' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/business-name-generator/">
+										{ translate( 'Business Name Generator' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/logo-maker/">{ translate( 'Logo Maker' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/webinars/">{ translate( 'Daily Webinars' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/courses/">{ translate( 'WordPress Courses' ) }</a>
+								</li>
+								<li>
+									<a href="https://developer.wordpress.com/" data-is_external="1">
+										{ translate( 'Developer' ) }{ ' ' }
+										<span className="lp-link-chevron-external">{ translate( 'Resources' ) }</span>
+									</a>
+								</li>
+							</ul>
+						</div>
+						<div>
+							<h3>{ translate( 'Company' ) }</h3>
+							<ul>
+								<li>
+									<a href="https://wordpress.com/about/">{ translate( 'About' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/partners/">{ translate( 'Partners' ) }</a>
+								</li>
+								<li>
+									<a href="https://automattic.com/press/" data-is_external="1">
+										<span className="lp-link-chevron-external">{ translate( 'Press' ) }</span>
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/tos/">{ translate( 'Terms of Service' ) }</a>
+								</li>
+								<li>
+									<a href="https://automattic.com/privacy/" data-is_external="1">
+										{ translate( 'Privacy' ) }{ ' ' }
+										<span className="lp-link-chevron-external">{ translate( 'Policy' ) }</span>
+									</a>
+								</li>
+							</ul>
+						</div>
+					</div>
+					<div className="lpc-footer-subnav-container">
+						<div className="lp-footer-language">
+							<h2 className="lp-hidden">{ translate( 'Language' ) }</h2>
+							<div className="lp-language-picker">
+								<div className="lp-language-picker__icon"></div>
+								<div className="lp-language-picker__chevron"></div>
+								<select
+									className="lp-language-picker__content"
+									title={ translate( 'Change Language' ) }
+									onChange={ ( e ) => navigate( e.target.value ) }
+									defaultValue={
+										translate.localeSlug === 'en'
+											? `https://wordpress.com/plugins`
+											: `https://wordpress.com/${ translate.localeSlug }/plugins`
+									}
+								>
+									<option>{ translate( 'Change Language' ) }</option>
+									<option lang="es" value="https://wordpress.com/es/plugins">
```

Comment:
I think it'd be nice if we can have this use the current domain; otherwise it makes it hard to use it when testing or developing on different domains.